test/uk/gov/hmrc/cbcr/controllers/ReportingEntityDataControllerSpec.scala

```diff
@@ -110,7 +115,7 @@ class ReportingEntityDataControllerSpec extends UnitSpec with ScalaFutures with
     }
 
     "respond with a 400 if ReportingEntity in request is invalid" in {
-      val result = controller.save()(badFakePostRequest)
+      val result = controller.save()(badFakePostRequest).run()
```

Comment:
what does .run() do here?

Reply:
Typed Actions when provided with a body of a wrong type return an "accumulator", which needs to be run in order to return the Response.